cfme/exceptions.py

```diff
@@ -390,3 +390,9 @@ def __init__(self, collection, filter_key):
     def __str__(self):
         return 'Action on Collection ({}) requires a filter: ({})'\
                .format(self.collection, self.filter_key)
+
+
+class NeedleNotFoundInLog(CFMEException):
+    """
+    Raised when log doesnt't contain needle
```

Comment:
Required:
This should be an `inline` comment.

Reply:
@digitronik Thank you for the review! I've changed it to inline.